pom.xml

```diff
@@ -51,6 +51,7 @@
     <snappy-java.version>1.1.7.2</snappy-java.version>
     <spring.version>5.0.9.RELEASE</spring.version>
     <tika.version>1.18</tika.version>
+    <xerces.version>2.12.0</xerces.version>
```

Comment:
I've tried updating Jena, but the build/testsuite fails with both Jena 3.7 and 3.8. But you are right, I will remove the Xerces dependency from the main pom.xml and turn this into a cleaner and more specific dependency override.

Of course, help with getting fcrepo4 working with Jena 3.8 would be greatly appreciated, since I'm not really a Java developer but just a concerned sysadmin.

Reply:
If, in your travails on this issue here, you are able to get Fedora _runnable_ on jdk9/10/11, that would also be hugely useful to all concerned sysadmins out there. JDK 8 has only a few months of life left (after Jan 2019, no further security patches will be supplied), and last I checked, you can't build or run Fedora on any of the new JDKs.

Reply:
@hardfalcon I made a PR to your branch that bumps Jena to 3.8.  The problem relates to fcrepo4's support for RDF 1.0.   In a past release, it was discovered that upgrading Jena brought Fedora into compliance with RDF/1.1, which was a breaking change for some production systems.  So the decision was made to support RDF/1.0 for the sake of backwards compatibility, at least for the 4.7 maintenance branch.   In fact, xerces is still a dependency in the `jena-patch` module, as it is used to transform RDF/XML responses so that is conforms to the older serialization.

Jena 3.7+ entirely remove support for the `xsd:QName` literal type; something that is listed in the spec as something that `SHOULD NOT` be used.  While there is always a chance that someone out there is using Fedora with grotty RDF/XML that uses this construct, the benefits of upgrading might outweigh this technical breakage of backwards compatibility.